lib/stats/folder.go

```diff
@@ -15,6 +15,7 @@ import (
 type FolderStatistics struct {
 	LastFile LastFile  `json:"lastFile"`
 	LastScan time.Time `json:"lastScan"`
+	NextScan time.Time `json:"nextScan"`
```

Comment:
So I suggest you wedge it in here:

https://sourcegraph.com/github.com/hfichtenberger/syncthing@faf3a96db78f3a17449d3abbc0ec2c3dd8dab6a8/-/blob/cmd/syncthing/gui.go#L612

So just add a new method to the model m.NextScan("folder") which is potentially stored on the runner, and then returned in the summary.